pkg/crd/kubermatic/v1/constraint.go

```diff
@@ -44,10 +48,44 @@ type ConstraintSpec struct {
 	ConstraintType string `json:"constraintType"`
 	// Match contains the constraint to resource matching data
 	Match Match `json:"match,omitempty"`
-	// Parameters specifies the parameters used by the constraint template REGO
+	// Parameters specifies the parameters used by the constraint template REGO.
+	// It supports both the legacy rawJSON parameters, in which all the parameters are set in a JSON string, and regular
+	// parameters like in Gatekeeper Constraints.
+	// If rawJSON is set, during constraint syncing to the user cluster, the other parameters are ignored
+	// Example with rawJSON parameters:
+	//
+	// parameters:
+	//   rawJSON: '{"labels":["gatekeeper"]}'
+	//
+	// And with regular parameters:
+	//
+	// parameters:
+	//   labels: ["gatekeeper"]
+	//
 	Parameters Parameters `json:"parameters,omitempty"`
 }
 
+type Parameters map[string]interface{}
```

Comment:
Why not having a concrete type here?

Reply:
Because the Gatekeeper Constraints CRDs are created dynamically, and its Parameters can hold anything. 

https://open-policy-agent.github.io/gatekeeper/website/docs/howto/#constraints

So it can be something easy like in the example above or it can whole objects. So we cant make a concrete type